python/mujoco/rollout.py

```diff
@@ -69,12 +67,12 @@ def rollout(model: mujoco.MjModel,
     ValueError: bad shapes or sizes.
   """
   # skip_checks shortcut:
-  #   don't infer nroll/nstep
```

Comment:
I understand why we don't need the argument, but it still needs to be inferred, right?

For example, if I have a single initial state but multiple control sequences for that state, or a single state and control sequence, but multiple warmstart vectors.

Can you confirm that this is still supported? (maybe add a test for these if they do not already exist)

Reply:
Yes it is still supported, though when using the `skip_checks` shortcut the user must ensure all of these arrays have a leading dimension of `nroll`. This is what the non-`skip_checks` version of rollout does before calling into the bindings.

It seems there is only one test related to inferring `nroll`. It is `test_one_rollout_fixed_ctrl`. It tests the case of passing a single initial state and control in but with nroll state and sensordata vectors. I will add tests for some of the other cases.

Reply:
My point is that "#   don't infer nroll/nstep" is still a correct comment, is it not? nroll is not an argument of the python API but still an argument of the low level API and that is inferred when skip_checks is False. Right?

Reply:
Yes please, add that test, and then I think we're done

Reply:
You're right, the old comment is the correct comment. My thinking was along the lines of "`nroll` is not an argument therefore it cannot be an inferred argument". But the comment string is not specific to arguments.